enterprise/backend/src/metabase_enterprise/sandbox/models/group_table_access_policy.clj

```diff
@@ -67,19 +67,23 @@
        ;; `row-level-restrictions` middleware). So include `:type` and `:status-code` information in the ExceptionInfo
        ;; data so it can be passed along if applicable.
        (when-not table-col-base-type
-         (throw (ex-info (tru "Sandbox Cards can''t return columns that aren't present in the Table they are sandboxing.")
-                         {:type        qp.error-type/bad-configuration
-                          :status-code 400
-                          :new-column  col
-                          :expected    (mapv :name table-cols)
-                          :actual      (mapv :name result-metadata-columns)})))
+         (let [msg (tru "Sandbox Cards can''t return columns that aren''t present in the Table they are sandboxing.")]
```

Comment:
`aren''t` double apostrophe

Reply:
You have to use double apostrophes in `tru`/trs`. Single ones are for things that are supposed to be returned literally without being quoted or something like that. I put some error checking in the `trs`/`tru` macros that is supposed to catch this but it missed that one for some reason. I noticed it and fixed it here

Reply:
ah. cool. my bad. :)